observation_portal/proposals/templates/proposals/requestfailurelimit.txt

```diff
@@ -11,3 +11,5 @@ Request url: {{ detail_url }}
 Thank you,
 
 {{ organization_name }}
+
+* You received this message because you opted to have notifications enabled. You can disable notifications globally here: {{ observation_portal_base_url }}/accounts/profile, or individually by proposal here: {{ observation_portal_base_url }}/proposals/{{ request.requestgroup.proposal }}
```

Comment:
Does the asterisk do any formatting here?

Reply:
I don't believe so, just raw text. Do you think I should remove it?

Reply:
No I think it's good to delineate that message in some way